docs/uca_guide.md

```diff
@@ -2,6 +2,38 @@
 title: Editing instructions for UCAs
 ---
 
+!["Navigating repositories"](img/uca_guide1) 
```

Comment:
I _think_ it needs the file extension also, e.g. uca_guide1.jpg
I will merge and get back to you for future reference.